ui/apps/platform/src/Containers/AccessControl/accessControlPaths.ts

```diff
@@ -1,4 +1,3 @@
-import { History } from 'react-router-dom';
```

Comment:
This lonely deletion caught my eye.

It looks like `AccessControlContainerProps` is an orphan (expand to see unchanged code).